app/src/main/java/com/einmalfel/podlisten/Provider.java

```diff
@@ -137,6 +137,10 @@ public long getMaxAge() {
   private static HelperV1 helper;
```

Comment:
@einmalfel No technical reasons. Just resolving sonar issue with adding setters not to write in static members from instance methods. In this case I made setters `synchronized`.